src/Groups/GAPGroups.jl

```diff
@@ -640,14 +689,14 @@ julia> G = symmetric_group(3);
 
 julia> subgroup_reps(G)
 4-element Vector{PermGroup}:
- Group(())
- Group([ (2,3) ])
- Group([ (1,2,3) ])
- Group([ (1,2,3), (2,3) ])
+ Permutation group of degree 3 and order 1
+ Permutation group of degree 3 and order 2
+ Permutation group of degree 3 and order 3
+ Permutation group of degree 3 and order 6
```

Comment:
I've also experimented with changing how we print groups, following some of the suggestions in the HackMD. At first I did not show the order (because we don't want to trigger its computation just by printing; and it seems potentially confusing to only sometimes show it).

But even that is a bit bland to my liking, we really loose information compared to before.

Perhaps at least the "detailed mode" for permutation groups should still show generators?

Reply:
Looking at other structures in Oscar, showing the generators seems to be not usual. For example, polynomial rings print only the number of generators.

Reply:
Yea, and it started to really annoy me. So I would argue that detailed should certainly print the generators.

Reply:
@simonbrandhorst I tend to agree, feel free to open an issue suggesting this (or submit a PR)